### PR TITLE
style: prettier all non-mdx files according to newest rules 💄

### LINKED ISSUE #1547: Prettier entire code base
#1546 upgrades prettier and settings. We should re-run prettier on the entire code base to follow the new rules.

### PR DESCRIPTION
**Pull request by gsong**
Style changes only: prettier the entire repo based on new prettier version and config.

* I've confirmed that I can `yarn build` locally.
* I've confirmed that I can `yarn develop` locally.

Addresses #1547.

### CODE REVIEW

**Comment by gsong** [comment]
@josh-heyer @frago12 Not sure which one of you want to approve this.

**Review by frago12 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/propose-new-topic.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
 ## Summary
+
 A short description of what you'd like to see covered
 
 ## Background info
 
 What do you already know about the topic? What would you like to learn? What sort of things will this information help you to do?
 
-(If you're proposing this topic because you think it would benefit *someone else*, then summarize *their* background knowledge and needs)
+(If you're proposing this topic because you think it would benefit _someone else_, then summarize _their_ background knowledge and needs)
```
Comment (josh-heyer) [inline .github/ISSUE_TEMPLATE/propose-new-topic.md:17]: Is there a way to NOT do this (underscores for emphasis)? Changes a lot of files.

Comment (gsong) [reply .github/ISSUE_TEMPLATE/propose-new-topic.md:17]: Not that I can see: https://prettier.io/docs/en/options.html. I personally like the fact that Prettier is so opinionated so we avoid this type of bike shedding.

Comment (josh-heyer) [reply .github/ISSUE_TEMPLATE/propose-new-topic.md:17]: Nice long-term, just a lot of changes to read through :)

Comment (gsong) [reply .github/ISSUE_TEMPLATE/propose-new-topic.md:17]: Yeah, why I wanted to isolate this specific PR, pay the one time price. Not sure why they weren't caught before by husky (maybe that came later).

**Review by josh-heyer — CHANGES_REQUESTED** [review]
This can't be merged as-is - prettier doesn't understand our Admonition syntax, and mangles it. 

Compare:

- Before: https://github.com/EnterpriseDB/docs/blob/main/product_docs/docs/epas/12/epas_compat_spl/15_object_types_and_objects/02_object_type_components/01_object_type_specification_syntax.mdx?plain=1#L71
- After: https://github.com/EnterpriseDB/docs/blob/style/george/prettier-all-the-things/product_docs/docs/epas/12/epas_compat_spl/15_object_types_and_objects/02_object_type_components/01_object_type_specification_syntax.mdx?plain=1#L71

The difference can be severe when rendered, depending on the content of the admonition (those beginning with code blocks can end up devouring surrounding paragraphs for instance).

There are around 1400 uses of admonitions in the content currently; most, if not all, are affected.

**Comment by gsong** [comment]
@josh-heyer @frago12 This PR is not non-mdx files only. I've separated mdx reformatting to #1557.

Let me know if you feel this is safe to merge.

**Review by josh-heyer — APPROVED** [review]
Looks solid on scan through; didn't run into any issues with these while testing earlier. Can look at the split PR tomorrow.

**Comment by frago12** [comment]
Looks good to me too

**Pull request merged**